app/pages/SiloUtilizationPage.tsx

```diff
@@ -72,6 +74,12 @@ export function SiloUtilizationPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Metrics24Icon />}>Capacity &amp; Utilization</PageTitle>
+        <DocsPopover
+          heading="Utilization"
+          icon={<Metrics16Icon />}
+          summary="System metrics allow you to monitor the allocation and utilization of your resources across silos, including CPU, memory, and storage."
```

Comment:
Not to be completely hamstrung by the docs, but they don't yet mention quotas. That might be something to update here once we update the docs?

Reply:
Or, I dunno, this might work: `System metrics let you monitor utilization of CPU, memory, and storage, in contrast to the total quota available to you.`

Reply:
Yeah, something like that. I wanted to say "against quotas" but it's hard to say it because there is the quotas for each individual silo and then the sum of all the quotas. And then of course we intend to also add a true (virtual) total capacity.